tests/api.sipnet.xml

```diff
@@ -0,0 +1,46 @@
+<?xml version="1.0"?>
+<pecan>
+  <pfts>
+    <pft>
+      <name>temperate.coniferous</name> 
+    </pft>
+  </pfts>
+
+  <meta.analysis>
+    <iter>3000</iter>
+    <random.effects>FALSE</random.effects>
+    <threshold>1.2</threshold>
+    <update>AUTO</update>
+  </meta.analysis>
+
+  <ensemble>
+    <variable>NPP</variable>
+  </ensemble>
+
+  <sensitivity.analysis>
```

Comment:
Probably don't want your test performing complicated, time-consuming analyses such as this sensitivity analysis or the meta.analysis above

Reply:
Yeah...I can leave those out...no problems